docs/resources/device_subnet_routes.md

```diff
@@ -22,7 +22,10 @@ resource "tailscale_device_subnet_routes" "sample_routes" {
   routes = [
     "10.0.1.0/24",
     "1.2.0.0/16",
-    "2.0.0.0/24"
+    "2.0.0.0/24",
+    # Or configure as an exit node
+    "0.0.0.0/0",
+    "::/0"
```

Comment:
This makes this example unrealistic, since it does not quite make sense to announce 0/0 alongside more specific subnets.

May I suggest to add another `tailscale_device_subnet_routes` block here that would just have two 0/0 routes, providing an example of an exit node?

Reply:
Good point, I have adjusted the example accordingly